spec/system/candidate_interface/offers_and_withdrawals/candidate_withdraws_spec.rb

```diff
@@ -70,8 +70,8 @@ def and_i_click_the_withdraw_link_on_my_final_choice
   end
 
   def then_i_see_a_confirmation_page
-    expect(page).to have_content('Are you sure you want to withdraw this application?')
-    expect(page).to have_content('Any upcoming interviews will be cancelled.')
+    expect(page).to have_content("Once you have a total of 15 unsuccessful or withdrawn applications, you will not be able to apply for any more courses until October #{RecruitmentCycle.real_current_year}")
```

Comment:
This is the non continuous application flow. I did the bare minimum to get it passing since we are going to be removing this shortly.